documentation/api/io.strimzi.api.kafka.model.Rack.adoc

```diff
@@ -1,25 +1,62 @@
-Configures rack awareness to spread partition replicas across different racks.
-
+The `rack` option configures rack awareness.
 A _rack_ can represent an availability zone, data center, or an actual rack in your data center.
-By configuring a `rack` for a Kafka cluster, consumers can fetch data from the closest replica.
-This is useful for reducing the load on your network when a Kafka cluster spans multiple datacenters.
+The _rack_ is configured through a `topologyKey`.
+`topologyKey` identifies a label on Kubernetes nodes which contains the name of the topology in its value.
+An example of such label is `topology.kubernetes.io/zone` (or `failure-domain.beta.kubernetes.io/zone` on older Kubernetes versions) which contains the name of the availability zone in which the Kubernetes node runs.
+You can configure your Kafka cluster to be aware of the _rack_ in which it runs and enable some additional features such as spreading partition replicas across different racks or consuming messages from the closest replicas.
 
-To configure Kafka brokers for rack awareness, you specify a `topologyKey` value to match the label of the cluster node used by Kubernetes when scheduling Kafka broker pods to nodes.
+For more information about Kubernetes node labels, see {K8sWellKnownLabelsAnnotationsAndTaints}.
+Consult your Kubernetes administrator regarding the node label that represents the zone or rack into which the node is deployed.
 
-If the Kubernetes cluster is running on a cloud provider platform, the label must represent the availability zone where the node is running.
-Usually, nodes are labeled with the `topology.kubernetes.io/zone` label (or `failure-domain.beta.kubernetes.io/zone` on older Kubernetes versions),
-which can be used as the `topologyKey` value.
+=== Spreading partition replicas across racks
 
-The rack awareness configuration spreads the broker pods and partition replicas across zones, improving resiliency, and also sets a `broker.rack` configuration for each Kafka broker.
+When rack awareness is enabled, Strimzi will set `broker.rack` configuration for each Kafka broker.
 The `broker.rack` configuration assigns a rack ID to each broker.
```

Comment:
Would this be better in the first section? And does this mean we can enable without further config?

Reply:
Ok, good point. I changed it to _configured_.